components/composition/Hidden/Hidden.css

```diff
@@ -0,0 +1,50 @@
+/*
+  Prettier tried to put everything on new lines, but we don't want that in this file.
+  Can't see how to disable just that rule.
+*/
+/* stylelint-disable prettier/prettier */
```

Comment:
Let it go to a new line, or add an extra prettier config.

We don't need to waste time on individual opinions about whether stuff should go to a new line because we have these tools to remove that decision for us.

If the config changes in the future it'll auto correct, but not if we disable it.